src/contexts/globalLayout/GlobalNavbar/LoggedInNav.tsx

```diff
@@ -48,6 +48,8 @@ function LoggedInNav({ queryRef }: Props) {
     queryRef
   );
 
+  // FIXME: GraphQL work goes here to get collection ID, if any (so that we can render separate options in navbar)
```

Comment:
Here is where I imagined the query would go (or combined with the above query)

Reply:
should be able to get collection ID like this:

```js
const { query } = useRouter()
const { collectionId } = query

// if collectionId is present, it means that we're on a collection page
```